### ISSUE TITLE
Release v1.6 sign-off

### ISSUE THREAD

**Issue by mrpollo** [PX4/Firmware#7258]
Is your team ready for a release?

* [ ] Project / Founder - @LorenzMeier
* [ ] Communication Architecture - @bkueng , @julianoes 
* [ ] UI / UX - @dogmaphobic, @DonLakeFlyer  
* [ ] Multicopter Flight Control - @Stifael, @MaEtUgR
* [ ] VTOL Flight Control - @tumbili, @AndreasAntener, @sanderux @bresch
    * [x] VTOL standard
    * [ ] VTOL tailsitter
    * [ ] VTOL tiltrotor
* [x] Fixed Wing Flight Control - @dagar, @priseborough 
* [ ] Racers - @anton-matosov 
* [ ] OS / drivers - @davids5 
* [x] UAVCAN / Industrial - @pavel-kirienko 
* [ ] State Estimation - @jgoppert, @priseborough 
* [x] VIO - @mhkabir, @ChristophTobler 
* [ ] ~Obstacle Avoidance - @vilhjalmur89~ - not implemented in mainline codebase yet.
* [ ] Snapdragon - @mcharleb 
* [x] Intel Aero - @lucasdemarchi, @simonegu 
* [x] Raspberry Pi / Navio - @bkueng 
* [x] Parrot Bebop - @eyeam3 
* [ ] Airmind MindPX / MindRacer - @iZhangHui 
* [x] Test team - @d3z0rian, @mrpollo

**Comment by jgoppert** [comment]
@priseborough how is EKF2 doing. I think LPE is stable.

**Comment by davids5** [comment]
There was a fix in upstream Nuttx this morning stm32_serial: fix freezing serial port I will need to back port. ASAP.

 

David

**Comment by dagar** [comment]
Could someone comment on the state of Pixhawk 3 support in v1.6.0? @klopezal, @sanderux

**Comment by mhkabir** [comment]
@dagar It is supported fully. We are waiting for fresh/fixed v1.3 hardware before we can polish the experience more. We are also extensively test flying it on a VTOL config.

**Comment by dagar** [comment]
Thanks @mhkabir. Could you do a quick pass and see if any of these are still relevant? https://github.com/PX4/Firmware/issues?q=is%3Aopen+is%3Aissue+label%3Apixhawk3

**Comment by mhkabir** [comment]
All of those issues have hardware fixes - but we don't have latest hardware yet, so they're still all relevant, yes.

**Comment by eyeam3** [comment]
PX4 on the Parrot Bebop is not running properly at the moment, but it should not gate the release.

**Comment by priseborough** [comment]
EKF2 has a PR pending for miscellaneous updates https://github.com/PX4/Firmware/pull/7252. 

Items 5 and 6 close vulnerabilities, and item 1 adds an important logging message for diagnostics, the others enable a 'nice to have' feature . I can split the 'nice to have' into a separate PR if required.

**Comment by davids5** [comment]
>There was a fix in upstream Nuttx this morning stm32_serial: fix freezing serial port I will need to back port. ASAP.

PR is https://github.com/PX4/Firmware/pull/7260

**Comment by priseborough** [comment]
I have closed the pending PR and split it into the robustness improvements that I think should go into the release:

https://github.com/PX4/Firmware/pull/7265

And the nice to have:

https://github.com/PX4/Firmware/pull/7264

**Comment by bkueng** [comment]
I think this one should go in: https://github.com/PX4/Firmware/pull/7226

**Comment by LorenzMeier** [comment]
I flagged all PRs with the milestone that I think should still go in.
https://github.com/PX4/Firmware/milestone/22

**Comment by nicolaerosia** [comment]
navio_gpio is broken on RPI3, see https://github.com/PX4/Firmware/pull/7241

**Comment by DonLakeFlyer** [comment]
FYI: I don't have time till tomorrow, then I'll do some SITL runs and sign off.

**Comment by sanderux** [comment]
@LorenzMeier should those pr's not go in before we test / sign off?

**Comment by nicolaerosia** [comment]
Maybe this should also go in https://github.com/PX4/Firmware/pull/7272

**Comment by LorenzMeier** [comment]
I'm closing this as we discussed this on the last dev call.
